script/openqa-validate-yaml

```diff
@@ -25,58 +25,53 @@ use 5.010;
 
 use FindBin '$RealBin';
 use lib "$RealBin/../lib";
+use Test::More;
 use Getopt::Long::Descriptive;
 use OpenQA::YAML qw(load_yaml validate_data);
 
 my ($opt, $usage) = describe_options(
     <<"EOM",
-$0 %o <yaml-file>
+$0 %o <yaml-files>
 
-Validate a YAML file against a schema.
-You can pass a YAML file or '-' for STDIN.
+Validate YAML files against a schema.
+You can pass YAML filenames or '-' for STDIN.
 EOM
     ['validate-schema', 'Validate Schema file itself also'],
-    ['schema-file=s',   "Schema file to validate against (default public/schema/JobTemplates-01.yaml)"],
-    ['help',            'print usage message and exit', {shortcircuit => 1}],
+    ['schema-file=s',   "Schema file or Url to validate against (default public/schema/JobTemplates-01.yaml)"],
```

Comment:
I would prefer `URL` here and single quotes.

Reply:
done